web/src/lib/components/shared-components/full-screen-modal.svelte

```diff
@@ -45,9 +45,13 @@
 
 <FocusTrap>
```

Comment:
Looks like the `FocusTrap` will have to be moved, to make sure that it's still receiving the keyboard events needed to manage focus within the modal. Making it a direct child of the `section` should fix this!

Reply:
@ben-basten Thank you, and I'll give it a try!

Reply:
I just changed this to just ignore events that originate from an input type that includes date and that seemed to fix the issue without needing to make any other changes.